classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,322 @@
+/**
+	String body = '';
+    String x = '\n\tpublic void method(){'
+            +'\n\t\tInteger i = 0;'
+            +'\n\t\ti = 1;'
+            +'\n\t\ti=2;'
+            +'\n\t\ti=3;'
+        +'\n\t}';
+    for(Integer i = 0; i < 200;i++){
+        body+=x.replace('method','method'+i);
+    }
+    system.debug('>>>\n\n'+body);
+ */
+public class ClaytonTestClass2 {
+    public void methodA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodC(){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.